### DISCUSSION TITLE
Win-64 v0.10.0 installer on conda-forge

### DISCUSSION THREAD

**Discussion by vincelim** [duckdb/duckdb#10852] [Ideas]
Are there plans to post the win-64 installer of DuckDB v0.10.0 on conda-forge? Looks like linux and osx installers were posted last week, but win-64 is still on v0.9.2. Link: https://anaconda.org/conda-forge/python-duckdb

**Comment by Mause** [comment]
We don't maintain the Conda feedstock, see here instead: https://github.com/conda-forge/python-duckdb-feedstock

**Comment by vincelim** [comment]
Ah, sorry about that. Thank you so much for pointing me to the right direction.
